Library/Homebrew/build.rb

```diff
@@ -83,7 +83,7 @@ def install
       ENV.keg_only_deps = keg_only_deps
       ENV.deps = formula_deps
       ENV.run_time_deps = run_time_deps
-      ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) }
+      ENV.x11 = reqs.any? { |rq| rq.is_a?(X11Requirement) || rq.is_a?(XorgRequirement) }
```

Comment:
One alternative fix is that `XorgRequirement` could inherit from `X11Requirement` just for the name alone, so that tests like this pass.